scalability/Makefile

```diff
@@ -220,16 +222,23 @@ core-experiment:
 	fi ; \
 	for program in $$EXPERIMENT_SET ; \
 	do \
-		for ext in $$EXPERIMENT_TYPE_SET ; \
-		do \
-			make $$program$$ext ; \
-		done ; \
+		if [ "${ENABLE_COVERAGE}" != "OFF" ] ; then \
```

Comment:
@rasoolmaghareh The adding of if ... fi is incorrect here because the enclosed loop is used simply for running the experiments, independent of whether the coverage computation is enabled or not.